eng/Packages.Data.props

```diff
@@ -108,7 +108,8 @@
     <!-- Azure SDK packages -->
     <PackageReference Update="Azure.AI.Inference" Version="1.0.0-beta.2" />
     <PackageReference Update="Azure.AI.OpenAI" Version="2.0.0" />
-    <PackageReference Update="Azure.AI.Projects" Version="1.0.0-beta.2" />
+    <PackageReference Update="Azure.AI.Projects" Version="1.0.0-beta.3" />
+    <PackageReference Update="Azure.AI.Agents" Version="1.0.0-alpha.20250211.4" />
```

Comment:
We should use a project reference for this.   We generally avoid dev feed packages here.   If you REALLY need to do this as a package reference, please use a local `VersionOverride`